### ISSUE TITLE
:inspect - sometimes the list on the left is scrolled up

### ISSUE THREAD

**Issue by sevu** [wesnoth/wesnoth#1940]
I'm not having a general explanation to this. In my case:
I use :inspect in the 2nd Tutorial. In this scenario are many events, the list is in fullscreen has a scrollbar.
Sometimes, when I scroll down and click an event, the list is scrolled up (like when pressing <pos1>).
It are not everytime the same events which make this happen.
I'm having a particular good chance to trigger this with the 5 post_advance events.

Another thing I noticed in this context, there is this blue background highlighting when selecting an entry or beeing mouseover over one. Sometimes this blue mouseover remains when you are already over another entry, or selected another.

**Comment by CelticMinstrel** [comment]
So basically what you're saying is... after clicking an event, the view is scrolled back up to the top?

This is probably caused by the contents of the event triggering a relayout or something. Maybe it could be fixed by scrolling to the just-clicked node after updating the text on the right... not sure.

**Comment by sevu** [comment]
Yes, thought this only happens to a few ones. And the events which trigger that behaviour don't do it every time.
Best would be to not scroll at all.

**Comment by CelticMinstrel** [comment]
Obviously, just not sure if that can be done if a relayout was triggered.

**Comment by sevu** [comment]
Maybe it shouldn't have been triggered.

As for the 2nd part of the bug report, as far as i could see so far it are the same events which potentially trigger the first part.

**Comment by Vultraz** [comment]
Was this related to #1979?

**Comment by sevu** [comment]
No, no changes here.


![downwards](https://user-images.githubusercontent.com/21158813/30402323-a3681e12-98dd-11e7-8d81-1c13e7d2ad92.png)

**Comment by Wedge009** [comment]
I checked this in 1.14.9. I don't know if it's a cause or related to this symptom but I notice that clicking on unique events such as victory or sighted (using an old save file of the last scenario of Delfador's Memoirs, sighted has all its code in one event block) then this does not happen. But non-unique events such as the multiple die and last_breath events, then I get the 'reset to scrolled up position' symptom.
